### ISSUE TITLE
NativeSelect component flashes first item in list before showing placeholder value.

### ISSUE THREAD

**Issue by lakebayagency** [svelteuidev/svelteui#175]
### What package has an issue

@svelteuidev/core

### A clear and concise description of what the bug is

When the page first loads, the NativeSelect component displays the first value in the options list briefly before then switching to the placeholder value.

In the attached video, I circle the component then move the mouse to refresh the page.  At about 7 seconds the page refreshes, and you can see this flash in action.

### In which browser(s) did the problem occur?

Chrome

### Steps To Reproduce

1. Refresh the page. :)

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

No

### Relevant Assets

![2022-08-03_20-23-09 (1)](https://user-images.githubusercontent.com/106922189/182757020-272d6ab2-b5c3-4eff-8e02-6ccd5fc2f3fc.gif)

```
        <NativeSelect 
            data={lead_types} 
            placeholder="Pick a project type"
            bind:value={lead_type}
        />
```

**Comment by Brisklemonade** [comment]
Thank you for raising this issue. This is the first time I've seen this happen, so I will need to do some investigation.

**Comment by Brisklemonade** [comment]
Fixed in (#177 ), changes will be pushed in the next patch!

**Comment by lakebayagency** [comment]
Thanks! :)
